server/tweetSearch.js

```diff
@@ -110,34 +113,41 @@ module.exports = function(client) {
         return tweetStore;
```

Comment:
Here it returns the tweetStore. I guess this line could return nothing too, I was just wondering if there was a reason to return it here but not if there are no new tweets added.